Untitled.ipynb

```diff
@@ -2,22 +2,22 @@
  "cells": [
```

Comment:
<p>why this change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leolavauclaravista/test-github/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>